### DISCUSSION TITLE
french translation

### DISCUSSION THREAD

**Discussion by SaiyajinK** [moudey/Shell#88] [General]
Hi,

I translated the entries into French.

1.7 : [shell_fr.zip](https://github.com/moudey/Shell/files/10220742/shell_fr.zip)

it would be nice to have an interface to easily manage the settings without going through a text editor.

++

**Comment by Eta-Beta** [comment]
Hi, congratulations, unfortunately I don't know the programming language and I had to change to Italian directly in the *nss files

**Comment by Elrick91** [comment]
How to use this French translation ? how to install it please?!
